src/Native/Unix/System.Native/pal_errno.cpp

```diff
@@ -8,8 +8,8 @@
 
 #include <errno.h>
 
-// ENODATA is not defined on FreeBSD.
-#if defined(__FreeBSD__)
+// ENODATA is not defined in FreeBSD 10.3 but is defined in 11.0
+#if defined(__FreeBSD__) & !defined(ENODATA)
```

Comment:
Does this need to be OS-specific?  I'm wondering if it can just be:
```C++
#ifndef ENODATA
#define ENODATA ENOATTR
#endif
```
or is ENODATA only equivalent to ENOATTR specifically on FreeBSD?